src/components/Settings/SettingsAddressbook.vue

```diff
@@ -71,11 +71,11 @@
 				<ActionCheckbox v-if="!toggleEnabledLoading"
 					:checked="enabled"
 					@change.stop.prevent="toggleAddressbookEnabled">
-					{{ enabled ? t('contacts', 'Enabled') : t('contacts', 'Disabled') }}
+					{{ t('contacts', 'Enabled') }}
 				</ActionCheckbox>
 				<ActionButton v-else
 					icon="icon-loading-small">
-					{{ enabled ? t('contacts', 'Enabled') : t('contacts', 'Disabled') }}
+					{{ t('contacts', 'Enabled') }}
```

Comment:
Should be Disable/Enable ? No?

Reply:
I'm not sure what the best UX is, and I'm definitely not a UX person, so I'd appreciate some further thoughts.

If the address book is disabled, I'd personally expect an "Enabled" checkbox to be unticked (`enabled == true`), or a "Disabled" checkbox to be ticked (`disabled == true`).

At present, if it's disabled, there is a "Disabled" checkbox that is unticked, which to me means `disabled == false`.

That's what I decided it would be best sticking with the same wording in both states and rely on the checkbox to convey the current state.

Reply:
I think we should move away from the preterit form. Use the action: "Enable/Disable" :)